hugo/layouts/partials/home/roadmap.html

```diff
@@ -1,10 +1,17 @@
-<h1>Our roadmap</h1>
-{{ range $r := $.Site.Data.home.roadmap.roadmap }}
-    <p>
-        {{ $r }}
-    </p>
-{{ end }}
-<p>Eiso Kant</p>
-<p>Co-Founder & Ceo</p>
+<div class="roadmap"> 
+    <h1>Our roadmap</h1>
+    {{ range $r := .roadmap }}
+        <p>{{ $r }}</p>
+    {{ end }}
+    <p>Eiso Kant</p>
+    <p>Co-Founder &amp; Ceo</p>
 
-{{ partial "home/our-links.html" . }}
+    <div class="roadmap__links">
```

Comment:
this is duplicated in other places. It could be used the deleted `home/our-links.html`